src/register/Register.jsx

```diff
@@ -0,0 +1,272 @@
+import React, { useState } from "react";
+import "./register.css";
+
+
+export default function Register() {
+    // set user data state
+    const [validName, setValidName] = useState(true);
+    const [validUsername, setValidUsername] = useState(true);
+    const [validEmail, setValidEmail] = useState(true);
+    const [validPassword, setValidPassword] = useState(true);
+    const [terms_and_conditions_checked, setTermsAndConditionsChecked] = useState(false);
+    const [need_mentoring, setNeedMentoring] = useState(false);
+    const [available_to_mentor, setAvailableToMentor] = useState(false);
+    const [responseMessage, setResponseMessage] = useState(null);
+
+
+    const handleSubmit = async e => {
+        e.preventDefault();
+        console.log(e.target)
+        let userInput = {}
+
+        new FormData(e.target).forEach((value, key) => {
+            console.log(key, " ", value);
+            if (key === "name")
+                userInput[key] = value;
+            else if (key === "username")
+                userInput[key] = value;
+            else if (key === "password")
+                userInput[key] = value;
+            else if (key === "email")
+                userInput[key] = value;
+            else if (key === "terms_and_conditions_checked") {
+                if (value === "true")
+                    userInput[key] = true;
+                else
+                    userInput[key] = false;
+            }
+            else if (key === "need_mentoring") {
+                if (value === "true")
+                    userInput[key] = true;
+                else
+                    userInput[key] = false;
+            }
+            else {
+                if (value === "true")
+                    userInput[key] = true;
+                else
+                    userInput[key] = false;
+
+            }
+        });
+
+
+        const payload = {
+            name: userInput["name"],
+            username: userInput["username"],
+            password: userInput["password"],
+            email: userInput["email"],
+            terms_and_conditions_checked: userInput["terms_and_conditions_checked"],
+            need_mentoring: userInput["need_mentoring"],
+            available_to_mentor: userInput["available_to_mentor"]
+        };
+        const requestRegister = {
+            method: "POST",
+            headers: {
+                "Accept": "application/json",
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify(payload)
+
+        };
+        let data = "";
+        fetch("http://127.0.0.1:5000/register", requestRegister)
+            .then(async response => {
+
+                data = await response.json();
+                console.log(typeof (data));
```

Comment:
Remove all unnecessary console logs we put for debugging

Reply:
yep, done 👍